lib/App/Config/Chronicle.pm

```diff
@@ -39,14 +40,19 @@ The configuration file is a YAML file. Here is an example:
           isa: Str
           default: "dummy@mail.com"
           global: 1
+        refresh:
+          description: "System refresh rate"
```

Comment:
Might it be useful to describe the units here too? The default is 10 - what does that mean?

Also looking at later code this appears to be a refresh *interval*, not rate. Larger numbers mean it refreshes less often, not more.

Reply:
This is just an example `yaml` file, these aren't actual fields on the package.